src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/IdentityResolution/ErrorAnalysis.java

```diff
@@ -0,0 +1,87 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.IdentityResolution;
+
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.DataSet;
+import de.uni_mannheim.informatik.dws.winter.model.MatchingGoldStandard;
+import de.uni_mannheim.informatik.dws.winter.model.Pair;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+import de.uni_mannheim.informatik.dws.winter.processing.Processable;
+import de.uni_mannheim.informatik.dws.winter.utils.WinterLogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ErrorAnalysis {
+	
+	private static final Logger logger = WinterLogManager.getLogger();
```

Comment:
You should use the static logger provided in IR_App. Just call it with `IR_App.logger.<method>`.